apps/andi/mix.exs

```diff
@@ -51,7 +51,7 @@ defmodule Andi.MixProject do
       {:placebo, "~> 1.2", only: [:dev, :test, :integration]},
       {:plug_cowboy, "~> 2.1"},
       {:simply_validate, ">= 0.2.0"},
-      {:smart_city, "~> 3.0.0", override: true},
+      {:smart_city, "~> 3.0", override: true},
```

Comment:
The current version of smart_city is 3.2, shall we consider updating it here?

Reply:
It will use 3.2.1